finagle-redis/src/main/scala/com/twitter/finagle/redis/SentinelClient.scala

```diff
@@ -26,7 +26,8 @@ object SentinelClient {
     val port: Int = props("port").toInt
     val runid: String = props("runid")
     val flags: Seq[String] = props("flags").split(",")
-    val pendingCommands: Int = props("pending-commands").toInt
+    val pendingCommands: Int = props.get("link-pending-commands")
```

Comment:
also, maybe we should call this `linkPendingCommands`  since that's its name going forward?  should be OK to break the API if you add a note to the CHANGES file.